transmart-rest-api/grails-app/controllers/org/transmartproject/rest/StudyQueryController.groovy

```diff
@@ -87,22 +88,25 @@ class StudyQueryController extends AbstractQueryController {
 
     /**
      * Study endpoint:
-     * <code>/v2/studies/studyIds?studyIds=</code>
+     * <code>/v2/studies/studyIds?studyIds=[]</code>
      *
-     * @param a list of the study names
+     * @param a json-encoded list of the study ids ${studyIds}
      *
-     * @return a list of the {@link org.transmartproject.db.i2b2data.Study} objects with names ${studyIds}
+     * @return a list of the {@link org.transmartproject.db.i2b2data.Study}
      * if all of them exist and the user has access; null otherwise.
      */
     def findStudiesByStudyIds(@RequestParam('api_version') String apiVersion) {
         if (params.studyIds == null) {
```

Comment:
Is it not possible to just specify `List<String> studyIds` as parameter of `findStudiesByStudyIds`?

Reply:
Never mind, Grails is a bit limited here (only path variable and command objects for the body seem to work).